Dockerfile.template

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2
+
+RUN set -ex \
+	&& mkdir -p /opt/changelog-generator \
+	&& apt-get update -yq \
+	&& buildDeps=' \
```

Comment:
I was able to get this gem and its dependencies to install without any of these `buildDeps`:

```diff
diff --git a/Dockerfile.template b/Dockerfile.template                                   
index 3e288b1..6ca7d52 100644               
--- a/Dockerfile.template                   
+++ b/Dockerfile.template                   
@@ -2,23 +2,12 @@ FROM ruby:2               
                                            
 RUN set -ex \                              
        && mkdir -p /opt/changelog-generator \                                           
-       && apt-get update -yq \             
-       && buildDeps=' \                    
-               libffi-dev \                
-               ruby-dev \                  
-               build-essential \           
-       ' \                                 
-       && apt-get install -y ruby \        
-       && apt-get install -y ${buildDeps} \                                             
-       && git clone -b %%GCG_VERSION%%-azavea https://github.com/azavea/github-changelog-generator \                                                                             
-               /opt/changelog-generator \  
+       && wget -qO- https://github.com/azavea/github-changelog-generator/archive/%%GCG_VERSION%%.tar.gz \                                                                        
+       | tar --strip-components=1 -xzC /opt/changelog-generator \                       
        && cd /opt/changelog-generator \
-       && gem update --system -N \
        && gem install -N bundler \
        && bundle install \
        && gem build github_changelog_generator \
-       && gem install -N *.gem \
-       && apt-get purge -y ${buildDeps} \
-       && rm -rf /var/lib/apt/lists/*
+       && gem install -N *.gem
 
-ENTRYPOINT ["github_changelog_generator"]
\ No newline at end of file
+ENTRYPOINT ["github_changelog_generator"]
```